run-dljc.sh

```diff
@@ -6,14 +6,16 @@
 ### Usage
 
 # - Move this script to an experiment directory.
```

Comment:
Change this to "link".  One doesn't want to move the script because that changes one's working directory.  One doesn't want to copy it, because then the experiments don't get the advantage of improvements.

Even better, remove this step.  The script should work if invoked from a different directory than it appears in.